components/infobox/wikis/apexlegends/infobox_league_custom.lua

```diff
@@ -139,6 +139,7 @@ function CustomLeague:defineCustomPageVariables()
 	Variables.varDefine('tournament_series2', _args.series2 or '')
 	Variables.varDefine('tournament_publisher', _args['ea-sponsored'] or '')
 	Variables.varDefine('tournament_pro_circuit_tier', _args.pctier or '')
+	Variables.varDefine('tournament_individual', _args.individual or '')
```

Comment:
Most team wikis use `_args.player_number` to determine if it's individual or not. So if `|player_number=` is set, individual, otherwise team.

Reply:
How do I edit this? Close this request and start a new one? Or can this be edited?

Reply:
I commited the new changes where "individual" is now dependent on "_args.player_number"